src/Library/demos/File Monitor/main.blp

```diff
@@ -0,0 +1,49 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.ToastOverlay overlay {
+  Adw.StatusPage {
+    title: _("File Monitor");
+    description: _("Monitors a file or directory for changes");
+    icon-name: "paper-symbolic";
+    Box {
+      orientation: vertical;
+      halign: center;
+      spacing: 18;
+
+      Label file_name {
+        label: _("");
```

Comment:
```suggestion
        label: _("");
```

This is a reserved string in gettext, you can't ever use `_("")`. I guess you can leave this `null`, or set it to `_("No File")`.